src/components/heatmap/Heatmap.js

```diff
@@ -161,25 +175,26 @@ const Heatmap = forwardRef((props, deckRef) => {
     }
   }, [expression, transpose]);
 
-  const [cellLabelMaxLength, geneLabelMaxLength] = useMemo(() => {
+  const [longestCellLabel, longestGeneLabel] = useMemo(() => {
     if (!expression) {
-      return [0, 0];
+      return ['', ''];
     }
+
     return [
-      max(expression.rows.map(cellId => cellId.length)),
-      max(expression.cols.map(geneId => geneId.length)),
+      expression.rows.reduce((a, b) => (a.length > b.length ? a : b)),
```

Comment:
What is `a` and `b`? could we use more explicit names?

Reply:
made more explicit and factored out